__tests__/Tooltip.test.js

```diff
@@ -50,4 +70,26 @@ describe('Tooltip component', () => {
       expect(e.message).toBe('No instances found with node type: "Triangle"');
     }
   });
+
+  it('does not render pointer when actionType is longPress', () => {
+    const component = TestRenderer.create(
+      <Tooltip
+        withPointer={false}
+        height={100}
+        width={200}
+        actionType="longPress"
+        popover={<Text>Info here</Text>}
+      >
+        <Text>Press me</Text>
+      </Tooltip>,
+    ).root;
+
+    component.findAllByType(TouchableOpacity)[0].props.onLongPress();
+    expect(component.instance.state.isVisible).toBe(true);
+    try {
+      component.findByType(Triangle);
+    } catch (e) {
+      expect(e.message).toBe('No instances found with node type: "Triangle"');
+    }
+  });
```

Comment:
Thanks for writing tests.

Reply:
> Hey @alveshelio, this PR is great man, I will take a look at it tomorrow and verify it in my test case app. If everything is fine I will merge. Thank you for the contribution.

Well, thank you very much for giving me the opportunity to contribute.